components/SwapForm/SwapFormStyles.tsx

```diff
@@ -3,10 +3,11 @@ import styled from 'styled-components'
 import { SegmentedController } from '../SegmentedController'
 
 export const SwapFormHeading = styled(Text).attrs(() => ({
-  type: 'heading',
+  type: 'title',
   variant: 'normal',
 }))`
   padding: 44px 0 32px;
+  margin: 15px;
```

Comment:
This is adding horizontal offset; is that intended? 
```suggestion
  margin: 15px 0;
```

Reply:
yup, its like this in the designs and I think it looks better